src/esp/core/CMakeLists.txt

```diff
@@ -38,4 +45,4 @@ target_link_libraries(core
 
 target_include_directories(core
   PUBLIC
-    ${PROJECT_BINARY_DIR})
+    "${PROJECT_BINARY_DIR}")
```

Comment:
Minor thing, but CMake doesn't behave like Bash in this case and correctly preserves spaces inside variables. So the quotes are unnecessary, the same as they would be unnecessary in the above `configure_file()`.